lib/generators/asset_target/USAGE

```diff
@@ -0,0 +1,10 @@
+Description:
+    Generate an a new "target" within our webpack assets setup.
+
+Example:
+    rails generate asset_target NAME
```

Comment:
Should we make this `asset-target` to encourage dasherizing the name, then?

I wonder if we can force dasherizing of `file_name` inside the `AssetTargetGenerator` too?